index.html

```diff
@@ -77,67 +78,76 @@ <h2>Examples of Usage</h2>
-      </section>
-      <section>
-        <h3>The <dfn data-link-for="canCheckInputPending">canCheckInputPending</dfn> method</h3>
-        <p>The <code>canCheckInputPending</code> method returns a value indicating if this api is supported in this environment and optionally if a specific InputType is supported. If the <code>canCheckInputPending</code> is called then the user agent MUST run the following steps:</p>
-        <ol>
-          <li>If <code>SupportedEventTypes</code> is empty return false.</li>
-          <li>If inputType is not equal to an empty string and if inputType is not in <code>SupportedEventTypes</code>, return false.</li>
-          <li>Return true.</li>
-        </ol>
+        <p class="note">The specification for the isInputPending method is intentionally written in such a way that a user agent may return false for any reason. This is so user agents may include their own security measures while implementing this api and remain compliant with this specification.</p>
       </section>
     </section>
-    <section data-link-for="Window" data-dfn-for="Window">
-      <h2>Extensions to the Window interface</h2>
+    <section>
+      <h2>Extensions to the <dfn data-cite="HTML5#the-window-object">Window</dfn> interface</h2>
       <pre class="idl">
         partial interface Window {
-          readonly attribute <dfn data-cite="is-input-pending#dom-scheduling">Scheduling</dfn> scheduling;
+          readonly attribute Scheduling scheduling;
         };
       </pre>
     </section>
     <section>
-      <h2><code>SupportedEventTypes</code></h2>
-      <p><code>SupportedEventTypes</code> is an internal array of event types that the user agent supports detecting pending events for. A user agent MUST not expose <code>SupportedEventTypes</code> to script.</p>
-      <p><code>SupportedEventTypes</code> MUST include:</p>
-      <ul>
-        <li><code><dfn data-cite="uievents#event-type-click">click</dfn></code></li>
-        <li><code><dfn data-cite="uievents#event-type-keydown">keydown</dfn></code></li>
-        <li><code><dfn data-cite="uievents#event-type-wheel">wheel</dfn></code></li>
-      </ul>
-      <p>SupportedEventTypes MAY include: Any <code><dfn data-cite="uievents#uievent">UIEvent</dfn></code>, <code><dfn data-cite="mediacapture-streams#event-summary">MediaCapture</dfn></code> Event or any other event type.</p>
+      <h2><dfn>ContinousEvents</dfn></h2>
+      <p>A task is considered to be a <code>ContinousEvent</code> if the task will dispatch an event that is a <a>UIEvent</a> or <a>TouchEvent</a> or a child of either and if for events of this type the user agent typically fires multiple events of this type is scission at a rate that is implementation and platform specific.</p>
```

Comment:
```suggestion
      <p>A task is considered to be a <code>ContinuousEvent</code> if the task will dispatch an event that is a <a>UIEvent</a> or <a>TouchEvent</a> or a child of either and if for events of this type the user agent typically fires multiple events of this type is scission at a rate that is implementation and platform specific.</p>
```

Reply:
Also, what does "is scission at a rate" mean?

Reply:
Another good catch, thanks! That's a typo, should be "in succession at a rate"